phpunit.xml.dist

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- http://phpunit.de/manual/4.1/en/appendixes.configuration.html -->
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.1/phpunit.xsd"
+         backupGlobals="false"
+         colors="true"
+         bootstrap="app/bootstrap.php.cache"
+>
+    <testsuites>
+        <testsuite name="Project Test Suite">
+            <directory>src/*/*Bundle/Tests</directory>
+            <directory>src/*/Bundle/*Bundle/Tests</directory>
+        </testsuite>
+    </testsuites>
+
+    <php>
+        <server name="KERNEL_CLASS_PATH" value="/app/TestKernel.php" />
```

Comment:
ApiTestCase allows you to use Kernel class directly. We need it, because we have two separated Kernels. The problem is with version of ApiTestCase on travis.